.github/workflows/ecosystem-batch-repo-check.yml

```diff
@@ -50,6 +49,6 @@ jobs:
                             --repo_url=${repositories[$i]} \
                             --branch_name=${{ steps.vars.outputs.pr_branch_name }} \
                             --tier=${tiers[$i]} \
-                            --token=${{ secrets.WORKFLOW_DISPATCH_PAT }} \
+                            --token=${{ secrets.GITHUB_TOKEN }} \
```

Comment:
From "[Using the GITHUB_TOKEN in a workflow](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow)":

> When you use the repository's GITHUB_TOKEN to perform tasks, events triggered by the GITHUB_TOKEN, with the exception of workflow_dispatch and repository_dispatch, will not create a new workflow run.

This means this action can't be triggered as `schedule`? I'm not sure this is a suitable workaround; or are you thinking we just trigger manually until we work out what the token problem is?

Reply:
Yeah we are fine the wf triggered is a `repository_dispatch` :)
https://github.com/qiskit-community/ecosystem/blob/9ff2fb7c54e2515010860e34f182bc64bb581a23/.github/workflows/ecosystem-project-check.yml#L12-L13

This means the GITHUB_TOKEN can't trigger anything else than `workflow_dispatch` and `repository_dispatch` so in our case we are good :)

Reply:
I let click `Resolve conversion` when you see my response ;)

Reply:
Ok I see, thanks